changelogs/fragments/474-yarn_fix-outdated-fix-list.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - yarn - Fixed an index out of range error when no outdated packages where returned by yarn executable (see https://github.com/ansible-collections/community.general/pull/474)
+  - yarn - Fixed an too many values to unpack error when scoped packages are installed (see https://github.com/ansible-collections/community.general/pull/474)
```

```suggestion
  - yarn - fixed an index out of range error when no outdated packages where returned by yarn executable (see https://github.com/ansible-collections/community.general/pull/474).
  - yarn - fixed an too many values to unpack error when scoped packages are installed (see https://github.com/ansible-collections/community.general/pull/474).
```
see the latest doc https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to